client/src/modules/layouts/Contacts/People/contactReducer.js

```diff
@@ -7,6 +7,10 @@ const contactsReducer = (state = [], action) => {
   switch (action.type) {
     case LOAD_CONTACTS:
       return action.payload;
+    case ADD_CONTACT:
+      let newContacts = [...state];
+      newContacts.push(action.payload);
```

Comment:
maybe `unshift()` here would be better cuz we will add new contact to the beginning of table in our view. Or even better to sort it by name before displaying using array's `sort()` by name cuz when we will have many contacts it will help a lot.